src/Microsoft.AspNetCore.Localization/CookieRequestCultureProvider.cs

```diff
@@ -118,5 +145,8 @@ public static ProviderCultureResult ParseCookieValue(string value)
 
             return new ProviderCultureResult(cultureName, uiCultureName);
         }
+
+        private static bool IsValidCulture(string cultureName) =>
+            CultureInfo.GetCultures(CultureTypes.SpecificCultures).Any(c => c.Name == cultureName);
```

Comment:
May be we can check against the supported cultures, and I think the check that I made is fine